src/main/java/cn/academy/crafting/api/ImagFusorRecipes.java

```diff
@@ -46,6 +46,14 @@ public void addRecipe(IFRecipe recipe) {
 		recipe.id = recipeList.size() - 1;
 	}
 	
+	public void removeRecipe(Item item) {
+		for (IFRecipe r : recipeList) {
+			if (r.consumeType.getItem() == item) {
```

Comment:
@WeAthFoLD Should here use `ItemStack`? Since people will probably use something that has damage value (or metadata, nbt tag, et cetera)...